.github/workflows/pr-ci.yaml

```diff
@@ -276,9 +276,6 @@ jobs:
   auto-commit-schemas:
     needs: [compare-graphql-schema, compare-openapi-schema, checkout-branch, update-schemas]
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      pull-requests: write
```

Comment:
The permissions shouldn't be removed - I'm assuming this is because of Atif's new PR to add them in.